cmd/ci-secret-bootstrap/vault.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/openshift/ci-tools/pkg/vaultclient"
+)
+
+var _ secretStoreClient = &vaultSecretStoreWrapper{}
```

Comment:
What's this for?

Reply:
It checks that the &vaultsecretStoreWrapper impelements the secretStoreClient interface